default_api_test.go

```diff
@@ -623,6 +623,7 @@ func TestDefaultApiService_CreateBranch(t *testing.T) {
 	type args struct {
 		projectKey     string
 		repositorySlug string
+		localVarPostBody   interface{}
```

Comment:
is it possible to add the test with actual postBody `{branch:"xyz"}` and confirm that we get that?

Reply:
Looks like it's not possible to add the test case mentioned above. Also, these test cases will not run as part of our test suite as it's related to the library only. Hence only fixed broken test in the previous commit.

Reply:
understood but vscode will run the unit test and make sure the code we have written works for example. 

ok if the client they use in tests doesn't support testing this specific case